docs/src/SUMMARY.md

```diff
@@ -24,6 +24,7 @@
     - [Contracts](./reference/experimental/contracts.md)
     - [Loop Contracts](./reference/experimental/loop-contracts.md)
     - [Concrete Playback](./reference/experimental/concrete-playback.md)
+    - [Bounded Non-deterministic variables](./reference/experimental/bounded_arbitrary.md)
```

Comment:
Why is this under the `Experimental features` section? These are features that currently require a `-Z` option.